packages/connect-extension-protocol/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@substrate/connect-extension-protocol",
-  "version": "0.3.0",
+  "version": "0.4.0",
```

Comment:
Should we not bump before releasing instead? It doesn't matter, but I'm personally used to bumping before releasing

Reply:
The reason is that `@substrate/connect-extension-protocol` is used from `@substrate/connect` and `@substrate/extension`, and I have to adapt them to the new API... It really feels wrong to declare in the `package.json` of `@substrate/connect` and `@substrate/extension` that their code depends on version `0.3.0` when the code is using an API that it's not compatible with that version... It's also a bit dangerous, because if when releasing the next versions of `@substrate/connect`/`@substrate/extension` we forgot to first release the  `@substrate/connect-extension-protocol`, then our build for those packages would work in this repo, we would be able to publish them, but the packages would be broken for the users as version `0.3.0` is not compatible with the new code...